### DISCUSSION TITLE
LlamaCPP with GPU Support

### DISCUSSION THREAD

**Discussion by SerenityNrrd** [TransformerOptimus/SuperAGI#965] [Ideas]
Allow us to use Custom Models like Gorilla or other llama2 models instead of OPENAI... or simply allow us to change the OpenAI base url

**Comment by neph1** [comment]
That is my goal with this fork: https://github.com/neph1/SuperAGI
It currently supports KoboldCpp (because that's what I use), but adding other backends should be trivial.
I haven't been able to keep it updated with all the changes from upstream, lately...

Here's the discussion:
https://github.com/TransformerOptimus/SuperAGI/discussions/669
